src/main/java/edu/kis/powp/jobs2d/visitor/IVisitor.java

```diff
@@ -0,0 +1,16 @@
+package edu.kis.powp.jobs2d.visitor;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+
+public interface IVisitor {
+    public void visit(DriverCommand command, Job2dDriver driver);
+}
```

Comment:
This isn't the way to create visitor! There should be separate methods for each type of DriverCommand. And why is there a driver as a param?